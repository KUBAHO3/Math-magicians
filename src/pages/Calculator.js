import React from 'react';
import MyCalculator from '../components/MyCalculator';
import '../styles/Calculator.css';

function Calculator() {
  return (
    <div className="container-calc">
      <h5 className="message">Let&apos;s do some math!</h5>
      <MyCalculator />
    </div>
  );
}

export default Calculator;
