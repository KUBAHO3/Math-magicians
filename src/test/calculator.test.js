import renderer from 'react-test-renderer';
import Calculator from '../pages/Calculator';

it('renders Quote page correctly', () => {
  const tree = renderer
    .create(<Calculator />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});