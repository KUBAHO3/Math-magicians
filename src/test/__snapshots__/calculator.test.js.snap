// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quote page correctly 1`] = `
<div
  className="container-calc"
>
  <h5
    className="message"
  >
    Let's do some math!
  </h5>
  <div
    className="calc-wrapper"
  >
    <div
      className="calc-display"
    >
      0
    </div>
    <div
      className="calc-row"
    >
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="cal-btn-special"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="calc-row"
    >
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="cal-btn-special"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="calc-row"
    >
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="cal-btn-special"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="calc-row"
    >
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="cal-btn-special"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="calc-row"
    >
      <button
        className="cal-btn-double"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="cal-btn-normal"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="cal-btn-special"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
